python/rpdk/java/resolver.py

```diff
@@ -1,10 +1,11 @@
-from rpdk.core.jsonutils.resolver import UNDEFINED, ContainerType
+from rpdk.core.jsonutils.resolver import MULTIPLE, UNDEFINED, ContainerType
 
 PRIMITIVE_TYPES = {
     "string": "String",
     "integer": "Integer",
     "boolean": "Boolean",
     "number": "Double",
+    MULTIPLE: "Object",
```

Comment:
If we can define a new resolved_type container type, it can be like:

```
    if resolved_type.container == ContainerType.MULTIPLE:
        return "Object"
```